projects/storefrontstyles/scss/components/checkout/steps/payment/_payment-method.scss

```diff
@@ -23,6 +23,15 @@
     }
   }
 
+  .cx-card-img-container {
+    .fab,
+    .fas {
+      &::before {
+        font-size: var(--cx-font-size, 66px);
```

Comment:
I feel like the icons were a bit smaller before. (like around 52px)

Reply:
the original size was 77x48. I'll dial it back to get the height to 48px.